packages/viewer/src/interactions/index.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @module Interactions
+ */
+export * from './interactionHandler';
+
+export * from './flyToPartKeyInteraction';
+
+export * from './interactionApi';
+
+export * from './interactionEvent';
+
+export * from './interactionHandler';
+
+export * from './keyInteraction';
+
+export * from './keyStateInteractionHandler';
```

Comment:
see https://github.com/Vertexvis/vertex-web-sdk/pull/120/files#r580383341

these should be all classes/interfaces used by the viewer class.